draft/spec/index.html

```diff
@@ -654,7 +654,9 @@ <h2>Manifest</h2>
                 of every content file in all versions of the <a>OCFL Object</a>. The value for each key
                 <span id="E092">MUST</span> be an array containing the <a>content path</a>s of files in the OCFL
                 Object that have content with the given digest. Content paths within a manifest block
-                <span id="E042">MUST</span> be relative to the <a>OCFL Object Root</a>.
+                <span id="E042">MUST</span> be relative to the <a>OCFL Object Root</a>. As JSON keys are case 
+                sensitive, while digests may not be, there is an additional requirement that each digest value 
```

Comment:
"may not be" --> "are not" since we specify that digests are case insensitive.

Reply:
Not all digests are ncessarily case insensitive and we don't actually require it as far as I can see. We just note that it is the case for the base16 digests that we currently specify.

Reply:
I think "may not be" is correct. Some digests might be case sensitive even if that doesn't apply to the ones we have specified so far. I would leave it as is.

Reply:
...but it IS awkward to validate if we can't assume that digests are case insensitive. Which would mean modifying the language around digests.

Reply:
IMO, every digest needs its own normalization scheme. That is what I built into my code